spec/src/main/asciidoc/microprofile-jwt-bridge-spec.asciidoc

```diff
@@ -17,3 +17,43 @@
 
 = Eclipse MicroProfile MP JWT Bridge
 
+## Mapping MP-JWT Tokens to Jakarta EE Container APIs
+
+The requirements of how a JWT should be exposed via the various Jakarta EE container APIs is discussed in this section.
+
+### Recommendations for Container Integration
+This section describes the expected behaviors for Jakarta EE container APIs other than JAX-RS.
+
+#### jakarta.security.enterprise.identitystore.IdentityStore.getCallerGroups(CredentialValidationResult)
+This method should return the set of names found in the `"groups"` claim in the JWT if it exists, an empty set otherwise.
+
+#### jakarta.ejb.SessionContext.getCallerPrincipal()
```

Comment:
@ayoho Adam, this text got me thinking, will it make it difficult for example, for `smallrye-jwt` to offer a simple MP JWT Bridge support to be activated with #13. I wonder if this text should remain optional for now. I'll need to investigate on our end